internal/clients/nightfall/nightfall.go

```diff
@@ -249,9 +272,20 @@ func (n *Client) ReviewDiff(ctx context.Context, logger logger.Logger, fileDiffs
 		}
 
 		// Determine findings from response and create comments
-		createdComments := createCommentsFromScanResp(cts, resp, n.DetectorConfigs)
+		createdComments := createCommentsFromScanResp(cts, resp, n.DetectorConfigs, n.TokenWhitelistMap)
 		comments = append(comments, createdComments...)
 	}
 
 	return comments, nil
 }
+
+func makeStrMap(strSlice []string) map[string]bool {
```

Comment:
is it worth just using `strset.Set` instead of creating my own set/map?

Reply:
yup definitely - less code for us to test/maintain :)